apps/browser/src/vault/fido2/content/page-script.ts

```diff
@@ -95,6 +95,11 @@ navigator.credentials.get = async (
   abortController?: AbortController
 ): Promise<Credential> => {
   const fallbackSupported = browserNativeWebauthnSupport;
+
+  if (options?.mediation && options.mediation !== "optional") {
```

Comment:
suggestion: simplify:)
```suggestion
  if (options?.mediation !== "optional") {
```

Reply:
I tried shortening the logic to `options?mediation !== "optional` however some sides don't pass the `mediation` keyword at all, and in those cases I believe well want to keep the `options?.mediation` conditional and skip throwing the error.